xmtp/src/conversation.rs

```diff
@@ -158,15 +159,20 @@ where
         )
         .store(&mut self.client.store.conn().unwrap())?;
 
+        let conversations = Conversations::new(&self.client);
+        if let Err(err) = conversations.process_outbound_messages().await {
```

Comment:
I could get on board with moving `process_outbound_messages` to `Client`. It's really not a "conversations" specific function (like `list` or `newConversation` are), so much as a general processing workload for messages.